runtime-config-linux.md

```diff
@@ -17,7 +17,7 @@ The following parameters can be specified to setup namespaces:
     * **`uts`** the container will be able to have its own hostname and domain name
     * **`user`** the container will be able to remap user and group IDs from the host to local users and groups within the container
 
-* **`path`** *(string, optional)* - path to namespace file
+* **`path`** *(string, optional)* - path to namespace file in the host mount namespace
```

Comment:
On Fri, Dec 18, 2015 at 01:48:35PM -0800, Vincent Batts wrote:
> > -* **`path`** *(string, optional)* - path to namespace file
> > +* **`path`** *(string, optional)* - path to namespace file in the host mount namespace
> 
> this seems more like a validation concern to me.

The spec is a meeting point beween runtime authors, and bundle
authors, runtime callers, and validation authors (who are likely both
bundle authors and runtime callers).  I don't see how you could have
an ambiguity that only impacted one subset.

> i'm not sure what you're technically attempting to achieve here.

I'm just trying to make sure folks don't misinterpret the namespace
path as “in the current mount namespace just before the setns call”,
which is what I did in ccons (and which you *want* for the alternative
namespace approach in [1]).  If we all agree that the interpretation
should be “paths are resolved in the mount namespace from which the
runtime is launched”, then landing wording to that effect seems easy
to do, and will either:

a. help reduce ambiguity and resulting conflicts, or
b. not take up too much space

depending on your view of how ambiguous the current wording is.

[1]: https://github.com/opencontainers/specs/pull/275/files#r48071357